os/arch/arm/src/amebad/amebad_reboot_reason.c

```diff
@@ -56,10 +56,15 @@
 #include <tinyara/config.h>
 
 #include <tinyara/reboot_reason.h>
+#include "ameba_soc.h"
 /****************************************************************************
  * Public functions
  ****************************************************************************/
 #ifdef CONFIG_SYSTEM_REBOOT_REASON
+
+#define UPPER_REBOOT_SYSTEM_WATCHDOG	BIT(0)
```

Comment:
How about define like below? Because it should be matched with common enum value in `tinyara/reboot_reason.h`
```
#define UPPER_REBOOT_SYSTEM_WATCHDOG BIT(REBOOT_SYSTEM_WATCHDOG)
```